tests/Aspire.Npgsql.EntityFrameworkCore.PostgreSQL.Tests/ConformanceTests.cs

```diff
@@ -3,23 +3,22 @@
 
 using Aspire.Components.Common.Tests;
 using Aspire.Components.ConformanceTests;
-using Microsoft.DotNet.RemoteExecutor;
-using Microsoft.EntityFrameworkCore;
+using Aspire.Npgsql.Tests;
 using Microsoft.EntityFrameworkCore.Internal;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
-using Npgsql;
 using Xunit;
 
 namespace Aspire.Npgsql.EntityFrameworkCore.PostgreSQL.Tests;
 
-public class ConformanceTests : ConformanceTests<TestDbContext, NpgsqlEntityFrameworkCorePostgreSQLSettings>
+public class ConformanceTests : ConformanceTests<TestDbContext, NpgsqlEntityFrameworkCorePostgreSQLSettings>, IClassFixture<PostgreSQLContainerFixture>
 {
     // in the future it can become a static property that reads the value from Env Var
-    protected const string ConnectionString = "Host=localhost;Database=test;Username=postgres;Password=postgres";
-
-    private static readonly Lazy<bool> s_canConnectToServer = new(GetCanConnect);
+    protected readonly PostgreSQLContainerFixture _containerFixture;
```

Comment:
```suggestion
    private readonly PostgreSQLContainerFixture _containerFixture;
```
? I don't see anyone using this.